lessons.json

```diff
@@ -1,24 +1,193 @@
+                            "icon" : "fa-question",
+                            "title" : "What Is Isidore?",
+                            "href" : "#"
+                        },
+                        {
+                            "icon" : "fa-plus",
+                            "title" : "Creating Course Sites",
+                            "href" : "#"
+                        }
+                    ]
+                },
+                {
+                    "header" : "Additional Articles",
+                    "references" : [
+                        {
+                            "icon" : "fa-external-link",
+                            "title" : "Isidore Log In Page",
+                            "href" : "https://isidore.udayton.edu"
+                        },
+                         {
+                            "icon" : "fa-external-link",
+                            "title" : "University of Dayton: Using NameCoach to Create an Environment of Inclusivity and Respect for Every Student",
+                            "href" : "https://name-coach.com/blog/case-study-using-namecoach-create-environment-inclusivity-respect-every-student/"
+                        },
+                        {
+                            "icon" : "fa-external-link",
+                            "title" : "2-Factor Authentication at UD",
+                            "href" : "https://udayton.teamdynamix.com/TDClient/Requests/ServiceDet?ID=14394"
+                        }
+                    ]
+                }
+            ],
+            "aside2title" : "Potential Questions You May Have",
+            "aside2questions" : [
+                {
+                    "question" : "I don't have a UD account yet. How can I log in?",
+                    "answer" : "Please contact the IT Service Center to obtain your credentials. You can reach them at itservicecenter@udayton.edu or at 937-229-3888. If your credentials are 'in-the-works', contact eLearning to obtain 'guest' logins."
+                },
+                {
+                    "question" : "What browsers work best with Isidore?",
+                    "answer" : "We recommend Chrome and Firefox, though Edge and Safari are also supported."
+                },
+                {
+                    "question" : "Why is 2-factor authentication needed?",
+                    "answer" : "As a part of the campus community, you play a part in keeping our campus safe - this includes from cyber attacks. Your password is the easiest way for attacks on our systems to happen, so the additional step of approving log ins on a secondary device is a critical step for safety."
+                }
+            ],
+            "aside3title" : "Did you know?",
+            "aside3content" : "Our Isidore homagepage has looked mighty different over the years. See all of Isidore's looks: https://drive.google.com/file/d/1YkDqB6d2oNHEmxASZ1dErPHdlRmwJU9V/view?usp=sharing"
+        } 
+   ],
```

Comment:
No comma here and need to end this object with a curly brace before jumping into Isidore 101.